Source/BluetoothError.swift

```diff
@@ -88,7 +89,9 @@ extension BluetoothError: CustomStringConvertible {
             return "Characteristic read error has occured: \(err?.localizedDescription ?? "-")"
         case let .characteristicNotifyChangeFailed(_, err):
             return "Characteristic notify change error has occured: \(err?.localizedDescription ?? "-")"
-            // Descriptors
+        case let .characteristicStateChangedFailed(_, err):
+            return "Characteristic state change error has occured: \(err?.localizedDescription ?? "-")"
```

Comment:
Here we need add an information that this error occured during setting notify value.